tests/codebuild/run_integtest.sh

```diff
@@ -1,33 +1,58 @@
 #!/bin/bash
 
-source tests/codebuild/common.sh
-
 # TODOs
 # 1. Add validation for each steps and abort the test if steps fails
 # Build environment `Docker image` has all prerequisite setup and credentials are being passed using AWS system manager
+# crd_namespace is currently hardcoded here for cleanup. Should be an env variable set in the pipeline and .env file
+
+source tests/codebuild/common.sh
+crd_namespace=${NAMESPACE_OVERRIDE:-"test-namespace"}
 
 # Verbose trace of commands, helpful since test iteration takes a long time.
 set -x 
 
-# A function to delete cluster, if cluster was not launched this will fail, so test will fail ultimately too
-function cleanup {
-    # We want to run every command in this function, even if some fail.
-    set +e
-
+# A function to print the operator logs for a given namespace
+# Parameter:
+#    $1: Namespace of the operator
+function get_manager_logs {
+    local crd_namespace="${1-sagemaker-k8s-operator-system}"
     if [ "${PRINT_DEBUG}" != "false" ]; then
```

Comment:
nit: Invert logic - exit if "${PRINT_DEBUG}" == "false"

Reply:
Fixing, but why is that better?

Reply:
Code readability. https://www.itamarweiss.com/personal/2018/02/28/return-early-pattern.html